salt/modules/zabbix.py

```diff
@@ -44,13 +44,12 @@
 
 def __virtual__():
     '''
-    Only load the module if Zabbix server is installed
+    Check whether zabbix_server or zabbix_agent are installed.
     '''
-    if salt.utils.which('zabbix_server'):
+    if salt.utils.which('zabbix_server') or salt.utils.which('zabbix_agent'):
```

Comment:
If this is to be opened up to machines other than the zabbix server, you don't need this at all. The rest of the module seems to use salt's http interfaces, so it should run without needing zabbix utilities installed.